### ISSUE TITLE
[Enhancement] color of navbar (Home and About Us) blended with background on Adventure Awaits

### ISSUE THREAD

**Issue by ayun6** [info442-aut21/d-team#82]
On the Adventure Awaits page, the color of the nav bar and background of Your Adventure Awaits are both white, making it difficult to see the first two components of the nav bar. 
Environment: Chrome 95.0.4638.69 on MacOS Big Sur

<img width="1139" alt="Screen Shot 2021-11-22 at 9 58 01 AM" src="https://user-images.githubusercontent.com/72246542/142912037-ee8e8104-fc7e-486e-9b6b-8e386934e374.png">

**Comment by IamSuperB4** [comment]
Could not reproduce.

**Comment by IamSuperB4** [comment]
I originally read this issue as the image being on top of the nav bar and hiding it. Now I see this is a similar issue to #79 , the nav bar not having a solid background color, so I am labeling it as a duplicate since it was done after #79 . I can also understand how the creator of this issue thought this might be a new issue though since it involved a picture overlap and not text overlap
